tests/components/fritz/test_update.py

```diff
@@ -124,4 +124,4 @@ async def test_available_update_can_be_installed(
             {"entity_id": "update.mock_title_fritz_os"},
             blocking=True,
         )
-        assert mocked_update_call.assert_called_once
+        mocked_update_call.assert_called_once()
```

Comment:
I prefer using `Mock.call_count` to avoid the risk of using the wrong attribute, eg due to a typo, which will pass unless we spec the mock.

```suggestion
        assert mocked_update_call.call_count == 1
```

Reply:
mhhh ... using `assert_called_once()` seems to be quiet common (_used ~650 times in our tests_)

Reply:
Yeah, unfortunately.

Reply:
I think it should be easily to be replace with a simple search&replace regex (_sure, not in this PR_ 😁)
but just for a better understanding of the possible risk - do you have an example of such a typo?

Reply:
Here's an example:

```suggestion
        mocked_update_call.ssert_called_once()
```

Reply:
uhhh ... that's really nasty :hushed: maybe we should discuss this in the members channel and try to get rid of `assert_called_once()` in out tests?

Reply:
Maybe. It's a bit of an uphill battle since it's part of standard library. If there's a linter that could check it, sure.